app/models/page.rb

```diff
@@ -7,5 +7,6 @@ class Page < AbstractPage
   validates :filename, presence: true
   validates :resource, presence: true
   validates :position, presence: true, uniqueness: {scope: :resource}
+  validates :filename, presence: true, uniqueness: {scope: :resource}
```

Comment:
Should delete line 7

Reply:
Thanks, I just pushed a fix. I also reordered the validations so the two scope ones are together, I think that looks a bit nicer